### PR TITLE
Changing the color of the progress bar via color.ini

### PR DESCRIPTION
**Pull request by sanoojes**
As #99

### CODE REVIEW

File: color.ini [hunk 1]
```diff
@@ -4,3 +4,5 @@ subtext                               = F0F0F0
 shadow                                = 000000
 text                                  = FFFFFF
 button                                = 30bf63
+button-active                         = 30bf63
```
Comment (hbishere) [inline color.ini:7]: 2bc260

File: color.ini [hunk 2]
```diff
@@ -4,3 +4,5 @@ subtext                               = F0F0F0
 shadow                                = 000000
 text                                  = FFFFFF
 button                                = 30bf63
+button-active                         = 30bf63
+accent                                = 30bf63
```
Comment (hbishere) [inline color.ini:8]: 1db954

**Review by hbishere — COMMENTED** [review]
Maybe use previous values for defaults

File: color.ini (hunk 2 above)
Comment (hbishere) [inline color.ini:8]: button-active: 2bc260
accent: 1db954

Comment (sanoojes) [reply color.ini:8]: yea suree

**Pull request merged**